jwst/pipeline/calwebb_image2.py

```diff
@@ -126,6 +128,12 @@ def process_exposure_product(
         input = self.flat_field(input)
         input = self.photom(input)
 
+        # Resample individual exposures
+        result = self.resample(input)
+
+        # write out resampled exposure
+        self.save_model(result, suffix='i2d')
```

Comment:
Might need to put this in an IF-block and only call it if `result` exists. In the event a user skips the resample step, it won't exist.

Reply:
Latest commit addresses this last comment.